tests/templates/test_subroutines/test_qrom.py

```diff
@@ -41,6 +41,20 @@ class TestQROM:
     @pytest.mark.parametrize(
         ("bitstrings", "target_wires", "control_wires", "work_wires", "clean"),
         [
+            (
+                ["111", "101", "100", "110"],
+                [0, 1, 2],
+                [3, 4],
+                None,
+                False,
+            ),
+            (
+                ["111", "101", "100", "110"],
+                [0, 1, 2],
+                [3, 4],
+                None,
+                True,
+            ),
```

Comment:
Why these new cases are added?

Reply:
I was not previously testing cases without `work_wires`. I add this for safety